ArkEcosystem/laravel-ui
# fix: webpack compilation warnings

## Summary

https://app.clickup.com/t/p59gcp

If you see the details of the warning that is currently hidden it says something like "@charset must precede all other statements", the error is because the imported `@toast-ui/editor/dist/toastui-editor.css` inside the `markdown-editor.css` files have a `@charset` definition, (moving that import to the first line doesn't remove the warning)

So the only solution is to hide the warning 

This PR:
- Hides the warning on the terminal
- Even hidden that warning still I receive a toast notification on my mac so this PR also modifies the plugin in webpack.mix to suppress the warnings.

To make it easy to implement these changes I created a standalone plugin so we only need to import the plugin and the plugin will take care of everything related to the markdown: assets compilation and hide the warnings.

## To test

Merge this on the msq and arks PRs, remove the compiled `public/css/markdown-editor.css` (to ensure is created again) and recompile, the markdown should be working as always
- https://github.com/ArkEcosystem/marketsquare.io/pull/2165
- https://github.com/ArkEcosystem/ark.io/pull/973

## Checklist
